source/mir/random/flex/package.d

```diff
@@ -344,8 +354,7 @@ private S flexImpl(Pdf, S, RNG)
                         goto finish;
                     }
                 }
-                // common approximation
-                X = hat.y + (inverseAntiderivative(antiderivative(hatLx, c) + hat.slope * u, c) - hat.a) / hat.slope;
+                X = hat.inverse(inverseAntiderivative(u * hat.slope + antiderivative(hatLx, c), c));
```

Comment:
Comes from my [draft report](http://files.wilzbach.me/mir/reports/tinflex.pdf) on page 7.
As mentioned `inverse` is the generalization - please note that the formula for `X` doesn't change here.